src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -28,6 +28,8 @@ public RemoteJSRuntime(IOptions<CircuitOptions> options, ILogger<RemoteJSRuntime
             JsonSerializerOptions.Converters.Add(new ElementReferenceJsonConverter(ElementReferenceContext));
         }
 
+        public JsonSerializerOptions ReadJsonSerializerOptions() => JsonSerializerOptions;
```

Comment:
Note: the reason I didn't use a property-with-getter here is to avoid confusion with the same property on the base class. Other usages of `JsonSerializerOptions` should access the property from the base directly without going through some extra layer of getter.

Since this is entirely internal API I'd prefer optimizing for not accidentally making an extra method call over the elegance of its appearance.

Same in the two other equivalent cases below.

Reply:
Could you add a comment to this effect to this API? Also `new` is your friend

Reply:
Not using `new` is specifically what I was referring to with my comment above.